### PR TITLE
Remove need for second JAR

### PR DESCRIPTION
**Pull request by DaMatrix**
As it doesn't make any sense to have two CoreMods when both of them are needed, all this does is merge them both into one CoreMod, which removes the need of the second JAR.

### CODE REVIEW

**Comment by DaMatrix** [comment]
Don't know why the README.md commit is showing up here. welp

**Comment by BigBastard** [comment]
Its an issue with how Forge loads CoreMods. Inside the .jar is a META-INF file with the information of the CoreMod. this only supports 1 CoreMod per META-INF unfortunately, so merging them would only prevent one of the CoreMods from loading.

:/

**Comment by DaMatrix** [comment]
When simply running without the second JAR I can't see block updates on the airship.  With this I can, which leads me to believe that it loads them both with one JAR.

**Comment by BigBastard** [comment]
Alright, sorry for not explaining this well, but basically I didn't want to do this because there were still issues with the CoreMod missing transformations, seemingly at random.

Adding the 2nd CoreMod only worked by effectively doubling the chances that a method doesn't get missed, so it was only working by pure chance.

I finally figured it out though, it was an issue with me checking if a class was extending another class (Checking if 'EntityPig' extends 'Entity' for example), it had to do with the class names being obsfucated at the time of checking for inheritance.

TL;DR the second jar was a band-aid to a gaping wound, and I wanted a more permanent solution. Sorry for the lack of communication :/

**Comment by DaMatrix** [comment]
OK :D
